### DISCUSSION TITLE
I'm trying to add my local language translation which is Tagalog(PH)

### DISCUSSION THREAD

**Discussion by MastinGood** [benawad/dogehouse#1353] [Q&A]
But when I try to push to staging this modal pops up. What should I do? should I ask the owner for write access? btw this my very first time contributing to a project.
![image](https://user-images.githubusercontent.com/21983499/111865073-ce5def00-899f-11eb-8eb7-b1e1475c0257.png)

**Comment by imadill** [comment]
You need to fork the dogehouse repository and then do changes in your fork. Then open a pull request with your fork to the dogehouse repository. :)
